cmd/tasks/src/lib.rs

```diff
@@ -267,16 +277,19 @@ fn tasks(context: &mut humility::ExecutionContext) -> Result<()> {
             core.run()?;
         }
 
-        println!("system time = {}", ticks);
-
+        println!(
+            "system time = {}",
+            ticks
+                .map(|t| t.to_string())
+                .unwrap_or_else(|| "unknown".to_owned())
```

Comment:
I think `unknown` caused some confusion yesterday. `ticks` can only be `None` if we're working remotely (and is _always_ `None` in that case), right? Maybe something more descriptive here would be useful - `unavailable-via-net` or `unavailable-remotely` or somesuch?

Reply:
sure, I like `unavailable-via-net`